iterations/2.md

```diff
@@ -15,13 +15,13 @@ Discuss the design of your application with your advisor. You may need to revise
 # First Feature
 
 <small>
-You may need to revise parts of the [Design](/iterations/1#design), and even parts of the [Project Proposal](/iterations/0#project-proposal) as you develop the application.
+You may need to revise parts of the [Design](/iterations/1#design), and even parts of the [Project Proposal](/iterations/0#project-proposal) as you develop the application. Remember to update the documents as required.
 </small>
 
 <small>
 Submit the code base for of your application in the root directory of the `master` branch of your group’s repository at `https://github.com/jhu-oose/{{site.course}}-group-<identifier>`.
 </small>
 
-You must implement at least the first feature of your application from end to end. Typically this is one of the central features of your application, but it must also be something relatively simple, because it’s your first feature. The feature must exercise every part of your [Architecture](/iterations/1#architecture), including database, tests, API design, deployment, documentation, and so forth. You must work with your advisor early in the iteration to establish what’s a suitable first feature.
+You must implement at least the first feature of your application from end to end. Typically, this is one of the central features of your application, but it should be something relatively simple, because it’s your first feature. The feature must exercise every part of your [Architecture](/iterations/1#architecture), including database, tests, API design, deployment, documentation, and so forth. It is a good idea to work with your advisor **early** in the iteration to establish what’s a suitable first feature.
```

Comment:
I don’t like the changes. Going from “you must” to “it is a good idea” waters the message down, which is why you had to introduce the bold. I’d just revert this line to the original.

By the way, [we’re using contractions](https://www.plainlanguage.gov/guidelines/conversational/use-contractions/), so it should have been “it’s a good idea”, not “it is a good idea.” Also, note the [curly quote](https://practicaltypography.com/straight-and-curly-quotes.html) I put there when writing the contraction.

Reply:
Alright, I'll keep the **early** and let go of all other changes.